docs/config/js/spectre.js

```diff
@@ -63,4 +63,49 @@ window.onload = function(){
         $(this).html( $(this).html().replace(/implementation defined/g,
             "implementation defined".italics()) );
     });
+
+    // Show popovers for references
+    $("body").children().find('a[href*="citelist.html#CITEREF_"]').each(function () {
+        var tooltip_base = $(this);
+        // Get the reference id, e.g. `CITEREF_Kopriva`. This is the one used as
+        // anchor on the bibliography page.
+        var ref_id = $(this).attr('href').match(/CITEREF_([a-zA-Z0-9]+)/)[0];
+        // Load the bibliography page to retrieve the reference data as nicely
+        // formatted HTML.
+        // This does not work locally because it is forbidden to access files,
+        // so just spin up a web server in the `html` directory with
+        // `python3 -m http.server` to enable this functionality
```

Comment:
can you add something about this command also in docs/DevGuide/Travis.md in the subsection on `How to perform the checks locally` in the `Documentation` item.

Reply:
done